addons/mail/static/src/models/message_action_view.js

```diff
@@ -0,0 +1,216 @@
+            }
+
+            if (isDeviceSmall) {
+                paddingClassNames.push('py-3')
+            } else {
+                paddingClassNames.push('py-2')
+            }
+    
+            return paddingClassNames.join(' ');
+        },
+        /**
+         * @private
+         * @returns {integer|FieldCommand}
+         */
+        _computeTabindex() {
+            if (this.messageAction.messageActionListOwnerAsReaction) {
+                return clear();
+            }
+            return 0;
+        },
+        /**
+         * @private
+         * @returns {string|FieldCommand}
+         */
+        _computeTitle() {
+            switch (this.messageAction.messageActionListOwner) {
+                case this.messageAction.messageActionListOwnerAsDelete:
+                    return this.env._t("Delete");
+                case this.messageAction.messageActionListOwnerAsEdit:
+                    return this.env._t("Edit");
+                case this.messageAction.messageActionListOwnerAsMarkAsRead:
+                    return this.env._t("Mark as Read");
+                case this.messageAction.messageActionListOwnerAsReaction:
+                    return this.env._t("Add a Reaction");
+                case this.messageAction.messageActionListOwnerAsReplyTo:
+                    return this.env._t("Reply");
+                case this.messageAction.messageActionListOwnerAsToggleCompact:
+                    return this.env._t("Compact");
+                case this.messageAction.messageActionListOwnerAsToggleStar:
+                    return this.env._t("Mark as Todo");
+                default:
+                    return clear();
+            }
+        },
+    },
+    fields: {
+        /**
+         * States the reference to the action in the component. Used in `Reaction` action
+         */
+        actionRef: attr(),
+        actionViewCounterContribution: attr({
+            default: 1,
+            readonly: true,
+        }),
+        ariaPressedState: attr({
+            compute: '_computeAriaPressedState',
+        }),
+        /**
+         * States t-att-class that changes according to the fields of the model.
+         */
+        classNames: attr({
+            default: '',
+            compute: '_computeClassNames',
+        }),
+        deleteConfirmDialog: one('Dialog', {
+            inverse: 'messageActionViewOwnerAsDeleteConfirm',
+            isCausal: true,
+        }),
+        messageAction: one('MessageAction', {
+            identifying: true,
+            inverse: 'messageActionView',
+        }),
+        /**
+         * Determines the reaction popover that is active on this message action list.
+         */
+        reactionPopoverView: one('PopoverView', {
+            inverse: 'messageActionViewOwnerAsReaction',
+            isCausal: true,
+        }),
+        paddingClassNames: attr({
+            default: '',
+            compute: '_computePaddingClassNames',
+        }),
```

Comment:
alphabetical order (should be in-between `messageAction` and `reactionPopoverView`)